get_coverage_for_challenge.sh

```diff
@@ -12,7 +12,7 @@ JACOCO_TEST_REPORT_XML_FILE="${SCRIPT_CURRENT_DIR}/build/reports/jacoco/test/jac
 mkdir -p ${SCRIPT_CURRENT_DIR}/target
 JAVA_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/target/java-code-coverage.txt"
```

Comment:
The coverage info should be written into a different file. See test output for more info on what the expectation is.

Reply:
Has this been changed? I know this worked previous, have you got this to work after your refactoring?